packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx

```diff
@@ -357,9 +229,8 @@ const SavedPage = ({
           userId={account ? account.user_id : 0}
         />
       ),
-      <div className={styles.albumsWrapper} key='albums'>
-        <AlbumsTabContent />
-      </div>
+      <AlbumsTabPage key='albums' />,
+      <PlaylistsTabPage key='playlists' />
```

Comment:
double checking, does this need to be behind a flag?

Reply:
yeah probably wise to go through a qa run first :)